contrib/build-push-ceph-container-imgs.sh

```diff
@@ -92,6 +99,7 @@ function push_ceph_imgs_latests {
       docker push ceph/$i:"$latest_name"
     done
   done
+  build_and_push_latest_bis
```

Comment:
I don't really like having a build + push inside a push() function.
Please move it elsewhere to keep a better semantic.

If you need to be here for any particular reason, it have to be explicit.